resources/install.rb

```diff
@@ -0,0 +1,161 @@
+unified_mode true
+
+property :version, String,
+  default: lazy { php_version },
+  description: 'Version of the php package to install.
+Defaults to platform specific numbers, see libraries/helpers.rb'
+
+property :configure_options, Array,
+  default: lazy { php_configure_options },
+  description: 'Array of options to be passed to the PHP configure script.
+Defaults to platform specific options, see libraries/helpers.rb'
+
+property :src_deps, Array,
+  default: lazy { php_src_deps },
+  description: 'Array of dependencies required to build PHP from source.
+Defaults to platform specific packages, see libraries/helpers.rb'
+
+property :src_recompile, [true, false],
+  default: lazy { php_src_recompile },
+  description: 'Boolean flag to indicate if PHP should be recompiled from source.
+Defaults to false'
+
+property :bin, String,
+  default: lazy { php_bin },
+  description: 'Path to the PHP executable.
+Defaults to platform specific paths, see libraries/helpers.rb'
+
+property :url, String,
+  default: lazy { php_url },
+  description: 'URL to download the PHP source code from.
+Defaults to platform specific URLs, see libraries/helpers.rb'
+
+property :checksum, String,
+  default: lazy { php_checksum },
+  description: 'Checksum of the PHP source code archive.
+Defaults to platform specific checksums, see libraries/helpers.rb'
+
+property :ext_dir, String,
+  default: lazy { php_ext_dir },
+  description: 'Directory where PHP extensions should be installed.
+Defaults to platform specific directories, see libraries/helpers.rb'
+
+property :conf_dir, String,
+  default: lazy { php_conf_dir },
+  description: 'Directory where PHP configuration files should be stored.
+Defaults to platform specific directories, see libraries/helpers.rb'
+
+property :ext_conf_dir, String,
+  default: lazy { php_ext_conf_dir },
+  description: ' Directory where PHP extension configuration files should be stored.
+Defaults to platform specific directories, see libraries/helpers.rb'
+
+property :php_packages, Array,
+  default: lazy { php_packages },
+  description: 'Array of names of the PHP Packages to be installed.
+Defaults to platform specific names, see libraries/helpers.rb'
+
+action :install do
+  if platform_family?('rhel', 'amazon')
+    include_recipe 'yum-remi-chef::remi'
+  elsif platform?('ubuntu')
+    include_recipe 'ondrej_ppa_ubuntu'
+  elsif platform?('debian')
+    apt_repository 'sury-php' do
+      uri 'https://packages.sury.org/php/'
+      key 'https://packages.sury.org/php/apt.gpg'
+      components %w(main)
+    end
+  end
+  include_recipe 'php::package'
```

Comment:
This shouldn't exist since we're migrating this to a resource. Should be replaced to whatever we have in that recipe (within reason).